lib/Feature/FeatureModelTransaction.cpp

```diff
@@ -36,6 +40,86 @@ void removeFeature(FeatureModel &FM,
   Trans.commit();
 }
 
+bool fvIsLeave(FeatureModel &FM, detail::FeatureVariantTy &FV) {
+  Feature *ActualFeature;
+  std::visit(
+      Overloaded{[&ActualFeature, &FM](Feature *F) { ActualFeature = F; },
+                 [&ActualFeature, &FM](string &F) {
+                   ActualFeature = FM.getFeature(F);
+                 }},
+      FV);
+  return ActualFeature->isLeave();
+}
+
+// TODO: delete, if other approach is used
+/*bool hasSuccessorsAmongOthers(
+    FeatureModel &FM, detail::FeatureVariantTy &FV,
+    std::vector<detail::FeatureVariantTy>::iterator Begin,
+    std::vector<detail::FeatureVariantTy>::iterator End) {
+  Feature *ActualFeature;
+  std::visit(
+      Overloaded{[&ActualFeature, &FM](Feature *F) { ActualFeature = F; },
+                 [&ActualFeature, &FM](string &F) {
+                   ActualFeature = FM.getFeature(F);
+                 }},
+      FV);
+
+  if (ActualFeature->children().empty()) {
+    return false;
+  }
+
+  for (auto Child : ActualFeature->children()) {
+    if (std::find(Begin, End, Child) != End) {
+    }
+  }
+  return false;
+}*/
+
+void removeFeatures(FeatureModel &FM,
+                    std::vector<detail::FeatureVariantTy> FeaturesToBeDeleted,
+                    bool Recursive) {
+  removeFeatures(FM, FeaturesToBeDeleted.begin(), FeaturesToBeDeleted.end(),
+                 Recursive);
+}
```

Comment:
This impl is quite short, so it could be moved to the header and merged with the declaration.

Reply:
Done